14-draft.json

```diff
@@ -54,15 +54,15 @@
       "type": "object",
       "properties": {
         "spacenet": {
-          "description": "See the <a target=\"_blank\" href=\"https://spacefed.net/wiki/index.php/Category:Howto/Spacenet\">wiki</a>.",
+          "description": "See the <a target=\"_blank\" href=\"https://spacefed.net/index.php/Category:Howto/Spacenet\">wiki</a>.",
           "type": "boolean"
         },
         "spacesaml": {
-          "description": "See the <a target=\"_blank\" href=\"https://spacefed.net/wiki/index.php/Category:Howto/Spacesaml\">wiki</a>.",
+          "description": "See the <a target=\"_blank\" href=\"https://spacefed.net/index.php/Category:Howto/Spacenet\">wiki</a>.",
           "type": "boolean"
         },
         "spacephone": {
-          "description": "See the <a target=\"_blank\" href=\"https://spacefed.net/wiki/index.php/Category:Howto/Spacephone\">wiki</a>.",
+          "description": "See the <a target=\"_blank\" href=\"https://spacefed.net/index.php/Category:Howto/Spacenet\">wiki</a>.",
```

Comment:
Does spacephone even exist anymore? I can't find it on the page.

Reply:
It sounds like spacephone doesn't exist anymore:
https://twitter.com/miepermans/status/1249279220519514113

Reply:
Then let's remove it... Do you know something about the rest of spacefed?

Reply:
The rest of spacefed still exists.

Reply:
Not sure if you were aware, but as mentioned in #80, https://spacephone.org still is alive and kicking.
Try `dig 2.2.9.9.7.9.1.3.e164.spacephone.org -t NAPTR` to see it working.

Reply:
@xopr do you know what hackerspaces are actively using it?

If it's still used actively by multiple spaces, feel free to open a PR to re-add it to the spec!

Reply:
@dbrgn: sure, there are SpacePhone entries for ACKspace, Hack42, Bitlair, Project73, PiXeLBar and NURDspace.
There are also a couple of hackerspace pages mentioning it: https://hackalot.nl/Spacetelefoon https://ackspace.nl/wiki/Phone:%2B31979922

It might not be a lot of spaces, but that's also true for spacesaml and that is still part of the spec.

Note that there were talks of officially bringing over SpacePhone into the SpaceFED environment since both approaches are inter-hackerspace services that grow better the more spaces join its concept.
While talk were progressing, the SpaceFED wiki was taken down due to spam attacks and has yet to return. (Also, the lookup service happens via its own domain: _e164.spacephone.org_ which make it seem conceptually more isolated than it actually is.

How do you want to go about this? PR for both **14** (retroactively bringing it back since it was never gone) and **15-draft** or only for **15-draft** and breaking the backward compatibility in a way?

Reply:
In theory we don't update 14 anymore, and having the spacephone entry in a v14 endpoint is perfectly valid, so we could just add it to 15-draft and nothing would break. On the other hand, it would also be backwards compatible to add the optional spacephone field back to v14. I'd tend towards the first approach though.

What are your thoughts, @SpaceApi/core?